xml/System.Collections.Immutable/ImmutableArray`1+Builder.xml

```diff
@@ -461,7 +461,15 @@
       <Parameters />
       <Docs>
         <summary>Removes all items from the array.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+<xref:System.Collections.Generic.ICollection%601.Count%2A> must be set to 0, and references to other objects from elements of the collection must be released.
```

Comment:
Is this remark relevant to the user? Sounds like implementation details that for some reason were added to the remarks in the triple slash comments.

Reply:
Agreed, will remove.